### PR TITLE
add registerUseStatus for breeding

### PR DESCRIPTION
**Pull request by hammeWang**

### CODE REVIEW

File: contracts/TokenUse.sol [hunk 1]
```diff
@@ -248,8 +263,13 @@ contract TokenUse is DSAuth, ITokenUse, SettingIds {
...
     function removeUseAndCreateOffer(uint256 _tokenId, uint256 _duration, uint256 _price, address _acceptedActivity) public {
         require(tokenId2UseStatus[_tokenId].owner == msg.sender);
+        require(tokenId2UseStatus[_tokenId].user != address(0), "Object does not exist.");
+        require(!isObjectInUseStage(_tokenId));
+
+        if (currentTokenActivities[_tokenId] != address(0)) {
+            IActivity(currentTokenActivities[_tokenId]).tokenUseStopped(_tokenId);
+        }
 
-        removeTokenUse(_tokenId);
```
Comment (HackFisher) [inline contracts/TokenUse.sol]:
```
delete tokenId2UseStatus[_tokenId];
```

useStatus is not deleted.

File: contracts/TokenUse.sol [hunk 2]
```diff
@@ -190,6 +190,21 @@ contract TokenUse is DSAuth, ITokenUse, SettingIds {
         }
     }
 
+    function registerTokenStatus(uint256 _tokenId, address _owner, address _user, uint256 _startTime, uint256 _endTime, uint256 _price, address _acceptedActivity) public auth {
```
Comment (HackFisher) [inline contracts/TokenUse.sol]: what is the user here? the siring contracts?

Comment (hammeWang) [reply contracts/TokenUse.sol]: _user is the bidder

Comment (HackFisher) [reply contracts/TokenUse.sol]: Here, there is a new token status created, buy no token transferred in, it is not compliant with existing rules.

**Pull request merged**